src/cases/containers/case-list/case-list.component.ts

```diff
@@ -59,6 +59,7 @@ export class CaseListComponent implements OnInit, OnDestroy {
   public paginationSubscription: Subscription;
   public isVisible: boolean;
   public jurisdictions: Jurisdiction[];
+  public selectedCases: SearchResultViewItem[] = [];
```

Comment:
These should be stored in the store, as they're needed on the next page when the button is clicked.

Reply:
the selected case is not added to store, please review.